treesource.c

```diff
@@ -347,7 +347,10 @@ void dt_to_source(FILE *f, struct dt_info *dti)
 {
 	struct reserve_info *re;
 
-	fprintf(f, "/dts-v1/;\n\n");
+	fprintf(f, "/dts-v1/;\n");
+	if (any_fixup_tree(dti, dti->dt))
+		fprintf(f, "/plugin/;\n");
+	fprintf(f, "\n");
```

Comment:
I think this is correct, but I seems like a very indirect way of diong it.  We're taking the `__fixups__` node, distributing across the tree as a bunch of fixup markers, then scanning the entire tree for fixup markers to mark it as `/plugin/`.

I think a clear way to do this would be to set the `DTSF_PLUGIN` flag on dtb input if there is a (non empty) `__fixups__` or `__local_fixups__` node, then on dts output output the `/plugin/` tag based on that flag.

Reply:
I went with testing just for the existence of `__fixups__` and `__local_fixups__`. I think the effort to check for non-emptiness (which has a different semantic for the two subtrees) is overengineering that isn't worth being invested.